testing/data/DATA_LICENSES.md

```diff
@@ -20,6 +20,7 @@
 - iflamigm.3ds: VTK Data: BSD-3-Clause
 - IM-0001-1983.dcm: VTK Data: BSD-3-Clause
 - InterpolationTest.glb: glTF-Sample-Models: Public Domain
+- kloofendal_43d_clear_1k.exr: Polyhaven: Public Domain
```

Comment:
arent polyhaven licenses all CC0 ?

Reply:
The HDR file we are using in the tests is coming from polyhaven too. Should I change its license to cc0?

Reply:
> Our assets are all licensed as [CC0](https://creativecommons.org/publicdomain/zero/1.0/), which is effectively [Public Domain](https://en.wikipedia.org/wiki/Public_domain) even in jurisdictions that do not support the Public Domain.

Reply:
Yeah, lets update them all to CC0, no reason not to.